### PR TITLE
New package for jwt

### PR DESCRIPTION
**Pull request by CharlyJazz**

### CODE REVIEW

**Comment by CharlyJazz** [comment]
I created a mock for kafka in order to avoid setup a real kafka connection for just a couple of tests.

File: auth-microservice/Pipfile [hunk 1]
```diff
@@ -19,6 +19,7 @@ uvicorn = "*"
 python-dotenv = "*"
 bcrypt = "*"
 factory-boy = "*"
+auth-validator = {editable = true, path = "/Users/charlyjazz/Charlyjazz/Chat-Firebase-Apache-Camel/auth-validator"}
```
Comment (aradkdj) [inline auth-microservice/Pipfile:22]: This won't work outside of that specific install. To make it portable you should consider using the repo as the source instead. According to [this SO answer](https://stackoverflow.com/a/19516714) it should be something like this:
```suggestion
auth-validator = {editable = true, path = "git+https://github.com/CharlyJazz/Chat-Firebase-Apache-Camel.git/#egg=pkg&subdirectory=auth-validator"}
```

File: auth-microservice/app/tests/conftest.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import asyncio
-from typing import Dict
-
 import pytest
 
+from typing import Dict
```
Comment (aradkdj) [inline auth-microservice/app/tests/conftest.py:4]: With python 3.9 you can subscript the built-in `dict`, no need to import it from typing
```suggestion
```

Also replace the `Dict[str, str]` on line 55 with a `dict[str, str`

File: auth-microservice/app/tests/test_users.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 from typing import Dict
 
 from app.tests.factories import UserFactory
+from app.core.config import settings
 
 @pytest.mark.asyncio()
 async def test_get_users_200(client: AsyncSession, session: AsyncConnection, user_token_headers: Dict[str, str]):
```
Comment (aradkdj) [inline auth-microservice/app/tests/test_users.py:10]: Same as above, you can subscript native `dict`, use that instead
```suggestion
from app.tests.factories import UserFactory
from app.core.config import settings

@pytest.mark.asyncio()
async def test_get_users_200(client: AsyncSession, session: AsyncConnection, user_token_headers: dict[str, str]):
```

File: chat-microservice/app/core/config.py [hunk 4]
```diff
@@ -1,9 +1,11 @@
-from typing import Any, Dict, Optional
+import os
 
+from typing import Any, Dict, Optional
```
Comment (aradkdj) [inline chat-microservice/app/core/config.py:3]: Again, no need to import `Dict` on 3.9
```suggestion
from typing import Any, Optional
```

File: chat-microservice/app/tests/api/test_create_message_endpoint.py [hunk 5]
```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from sqlalchemy.ext.asyncio import AsyncSession
+from typing import Dict
```
Comment (aradkdj) [inline chat-microservice/app/tests/api/test_create_message_endpoint.py:4]: Again, no need to import `Dict` on 3.9
```suggestion
```

File: chat-microservice/app/tests/conftest.py [hunk 6]
```diff
@@ -1,24 +1,36 @@
 import pytest
 
+from jose import jwt
+from typing import Dict
```
Comment (aradkdj) [inline chat-microservice/app/tests/conftest.py:4]: Again, no need to import `Dict` on 3.9
```suggestion
```

File: chat-microservice/Pipfile [hunk 7]
```diff
@@ -22,6 +22,7 @@ factory-boy = "*"
 kafka-python = "*"
 aiokafka = "*"
 cassandra-driver = "*"
+auth-validator = {editable = true, path = "/Users/charlyjazz/Charlyjazz/Chat-Firebase-Apache-Camel/auth-validator"}
```
Comment (aradkdj) [inline chat-microservice/Pipfile:25]: Again, this won't work outside of that specific install. To make it portable you should consider using the repo as the source instead. According to [this SO answer](https://stackoverflow.com/a/19516714) it should be something like this:
```suggestion
auth-validator = {editable = true, path = "git+https://github.com/CharlyJazz/Chat-Firebase-Apache-Camel.git/#egg=pkg&subdirectory=auth-validator"}
```

**Review by aradkdj — CHANGES_REQUESTED** [review]

**Pull request merged**